app/views/coc/bereavement_transferor.scala.html

```diff
@@ -29,31 +29,31 @@
 mainConfig = views.helpers.MainConfig()
 ) {
 
-<a href="@controllers.routes.UpdateRelationshipController.history" class="link-back">@Html(Messages("generic.back"))</a>
+<a href="@controllers.routes.UpdateRelationshipController.history" class="link-back">@messages("generic.back")</a>
 
 <header class="page-heading">
-    <h1 class="heading-xlarge" id="bereavement-transferor">@Html(Messages("change.status.bereavement.sorry"))</h1>
+    <h1 class="heading-xlarge" id="bereavement-transferor">@messages("change.status.bereavement.sorry")</h1>
 </header>
 
-<p>@Html(Messages("change.status.bereavement.tr.instruction"))</p>
+<p>@messages("change.status.bereavement.tr.instruction")</p>
 
 <ul class="list list-bullet">
-    <li>@Html(Messages("change.status.bereavement.tr.instruction-1"))</li>
-    <li>@Html(Messages("change.status.bereavement.tr.instruction-2"))</li>
+    <li>@messages("change.status.bereavement.tr.instruction-1")</li>
+    <li>@messages("change.status.bereavement.tr.instruction-2")</li>
 </ul>
 
-<h2 class="heading-medium">@Html(Messages("change.status.bereavement.notify-hmrc"))</h2>
+<h2 class="heading-medium">@messages("change.status.bereavement.notify-hmrc")</h2>
 
-<p>@Html(Messages("hmrc.contact-details-2")) <br>
-@Html(Messages("hmrc.contact-details-3")) <br>
-@Html(Messages("hmrc.contact-details-4"))</p>
+<p>@messages("hmrc.contact-details-2") <br>
+@messages("hmrc.contact-details-3") <br>
+@messages("hmrc.contact-details-4")</p>
 
-<h3 class="heading-small">@Html(Messages("hmrc.contact-detials.time"))</h3>
-<p>@Html(Messages("hmrc.contact-details-5")) <br>
-@Html(Messages("hmrc.contact-details-6"))</p>
-<p>@Html(Messages("hmrc.contact-details-7"))</p>
+<h3 class="heading-small">@messages("hmrc.contact-detials.time")</h3>
+<p>@messages("hmrc.contact-details-5") <br>
+@messages("hmrc.contact-details-6")</p>
+<p>@messages("hmrc.contact-details-7")</p>
 
-<h3 class="heading-small">@Html(Messages("hmrc.contact-detials.calling.time"))</h3>
-@Html(Messages("hmrc.contact-details-8"))</p>
-<p>@Html(Messages("hmrc.contact-details-9"))</p>
+<h3 class="heading-small">@messages("hmrc.contact-detials.calling.time")</h3>
```

Comment:
Needs HTML tag as the welsh message includes a ascii code. Alternatively, replace the ascii code with its character equivalent

Reply:
Same for all instances of this message ID

Reply:
replaced ascii code